code/modules/mob/living/carbon/alien/humanoid/life.dm

```diff
@@ -16,3 +14,9 @@
 		temperature = max(loc_temp, temperature-change)
 	temp_change = (temperature - current)
 	return temp_change
+
+/mob/living/carbon/alien/humanoid/royal/queen/Life()
+	. = ..()
+	if(!has_gps && SSshuttle.emergency.mode == SHUTTLE_STRANDED)
```

Comment:
why do you keep putting shitty checks in life

Reply:
didnt even learn from rideable mobs smh

Reply:
Please handle this elsewhere, and not on a continuous check every life tick.
Send a signal when the emergency shuttle mode changes, for example.